packages/org.openapis.v3/Document.pkl

```diff
@@ -0,0 +1,79 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+open module org.openapis.v3.Document
```

Comment:
Should this be called `OpenAPI`, instead of `Document`?

The spec calls this the "OpenAPI object": https://spec.openapis.org/oas/v3.0.3#openapi-object

Reply:
A `Document` has exactly one `Object` representing it, and I think `Document` here fits better with the established terminology in the OpenAPI community.

Reply:
There seems to be two top-level types available in the spec: `Document`, and `Documents`, which is just a list of `Document`.